packages/sit-onyx/src/components/OnyxInfoTooltip/OnyxInfoTooltip.vue

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" setup>
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import circleInformation from "@sit-onyx/icons/circle-information.svg?raw";
+import OnyxTooltip from "../OnyxTooltip/OnyxTooltip.vue";
+import type { OnyxInfoTooltipProps } from "./types";
+import { injectI18n } from "../../i18n";
+
+const props = defineProps<OnyxInfoTooltipProps>();
+const { t } = injectI18n();
+</script>
+
+<template>
+  <OnyxTooltip
+    v-if="props.text"
+    class="onyx-info-tooltip"
+    :text="props.text"
+    :position="props.position"
+  >
+    <button :aria-label="t('showInfoTooltip')" class="onyx-info-tooltip__trigger">
+      <OnyxIcon :icon="circleInformation" size="12px" />
+    </button>
+  </OnyxTooltip>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers";
+
+.onyx-info-tooltip {
+  @include layers.component() {
+    &__trigger {
```

Comment:
the trigger seems to be larger than it should be:
![image](https://github.com/SchwarzIT/onyx/assets/67898185/d822e587-7692-4315-b172-cbc127323598)

Reply:
How did you find that?

Reply:
I selected the info tooltip with the dev tools in Storybook

Reply:
fixed